app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title><%= content_for?(:title) ? yield(:title) : "Untitled" %></title>
+
+    <%= stylesheet_link_tag    "application" %>
+    <%= javascript_include_tag "application", 'data-turbolinks-track' => true %>
+    <%= csrf_meta_tags %>
+  </head>
+
+  <body>
+
+      <header class="page-header row">
+
+        <h2>
+          <%= link_to "Media Ranker", root_path %>
+          <small>
+            Ranking the Best of Everything
+          </small>
+        </h2>
+
+        <nav>
+          <ul>
+            <li><%= link_to "View top media", root_path, class: "button small float-left" %></li>
+            <li><%= link_to "View all media", works_path, class: "button small float-left" %></li>
+            <li><%= link_to "Add a new work", new_work_path, class: "button small float-left" %></li>
+            <li><%= link_to "View all users", users_path, class: "button small float-left" %></li>
+            <div class="float-right">
+              <li>
+                <% if session[:user_id] %>
+                  <%= link_to "Logged in as #{User.find_by(id: session[:user_id]).name}", user_path(:user), class: "button tiny"  %>
+                  <%= link_to "Logout", logout_path, method: :delete, class: "button tiny" %>
+                <% else %>
+                  <%= link_to "Login", login_path, class: "button" %>
+                <% end %>
+              </li>
+            </div>
+          </ul>
+        </nav>
+
+      </header>
+
+    <section class="flash">
+      <% flash.each do |name, message| %>
+        <div class="<%= name %>">
+          <%= message %>
+        </div>
+      <% end %>
+    </section>
+
+    <%= yield %>
+
```

Comment:
It's often wise to wrap the `yield` in a `<main>` tag, since that's where the main content of your web page will appear.